plugins/services/src/js/service-configuration/PodContainerConfigSection.js

```diff
@@ -65,7 +65,7 @@ const PodContainerConfigSection = ({containerConfig, appConfig, onEditClick, ind
       </ConfigurationMapRow>
 
       {/* Resources */}
-      {fields.resources.cpus && (
+      {!!fields.resources.cpus && (
```

Comment:
TL;DR use `get<Section>` or extract private components that is. Let's ban this syntax.

... or just write those private components and put all the logic there. To me this discussion went to far already. I'd say we can use the syntax only and only if we have a boolean flag. Otherwise we always will put a type checking / casting function that would nullify all the beauty of the approach.